### DISCUSSION TITLE
feat(plugin/habits): respect skipped-repositories

### DISCUSSION THREAD

**Discussion by revilowaldow** [lowlighter/metrics#1144] [Ideas]
The habits plugin doesn’t seem to respect skipped repositories when generating facts. It’s thinking that I only use spaces and always commit on a Thursday afternoon because I just did a load of setting up this tool (which is great btw 😘).
I’ve got skipped repositories excluding my homepage repo but it still seems to be using those commits. 
The suggestion is to ignore any commits from the excluded repos list when calculating habits from commits

**Comment by lowlighter** [comment]
Hi ! 

This will be tracked in #1147, thanks a lot for your feedback and your support !
Will try to check what is causing this behaviour 👍
